src/main/scala/utils/Implicits/RichImplicits.scala

```diff
@@ -0,0 +1,21 @@
+package utils.Implicits
```

Comment:
Do you mind moving that file in the level above and using the package `com.gu.contentatom.renderer.utils`?

Reply:
:+1: